src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -363,7 +363,7 @@ protected function getQueryBuilderBetweenDatesGroupByDate(\DateTime $from, \Date
         $queryBuilder = $this->getEntityManager()->getConnection()->createQueryBuilder();
 
         return $queryBuilder
-            ->from('sylius_order', 'o')
+            ->from($this->getClassMetadata($this->_entityName)->getTableName(), 'o')
```

Comment:
It's not the one from entity manager but from the Connection itself. It's kind of an SQL query instead of DQL (I've tried entity name instead of table name it does not work).